common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -53,6 +53,16 @@ public abstract class AzureActiveDirectoryAudience {
     public static final String ALL = "common";
     public static final String MSA_MEGA_TENANT_ID = "9188040d-6c67-4c5b-b112-36a304b66dad";
 
+    // The values here are audience values reported in telemetry
+    // rather than the values used to send to eSTS.
+    public enum AudienceTelemetryConstant {
+        MSA_AUDIENCE_FOR_REPORTING,
+        COMMON_AUDIENCE_FOR_REPORTING,
+        AAD_AUDIENCE_FOR_REPORTING,
+        UNKNOWN_AUDIENCE_FOR_REPORTING;
```

Comment:
Since you are invoking `.name` you need to make sure you define the names here exactly how you'd like them in telemetry:

```suggestion
        MSA,
        COMMON,
        AAD,
        UNKNOWN_AUDIENCE;
```

Reply:
added